server/controllers/tripsControllers.js

```diff
@@ -1,8 +1,19 @@
 import validate from '../middlewares/validateTrip';
-import trips from '../models/tripsModels';
 import express from 'express';
 import response from '../helpers/response';
 const router = express.Router();
+//Database connection
+import {Client, Pool} from 'pg';
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
This code is very similar to that in `bookingControllers.js`. Find a way to make reusable methods instead of repeating code.